File: Swift-C++-Testsuite/test/linearConstantPropagation/swift/functiontestaddition.swift
func magical(z: Int) -> Int {
    let a = 78_092_178
    let b = 45124
    let x = a + b
    return x + z
}
let rand = Int.random(in: 1..<100)
var myResult = -1
if rand < 40 {
    myResult = magical(z: rand)
    // 78149714
} else {
    myResult = magical(z: 12411)
    // 78149713
}

File: AutoTranspiledTests/try-1/functiontestaddition.cpp

 #include <iostream>
 #include <random>
 
 int magical(int z)
 {
     int a = 78092178;
     int b = 45124;
     int x = a + b;
     return x + z;
 }
 
 int main()
 {
     std::uniform_int_distribution <int> rand(1, 100);
 
     int rand_num = rand(rng);
 
     int myResult = -1;
     if (rand_num < 40)
     {
         myResult = magical(rand_num);
         // 78149714
     }
     else
     {
         myResult = magical(12411);
         // 78149713
     }
     return 0;
 }